tests.py

```diff
@@ -885,6 +888,66 @@ def testDownloadDockerFile(self):
         self.assertEqual(expected_dockerfile, f.read())
 
 
+class TestDEVolumes(unittest.TestCase):
+  """Tests various volumes/bind mounts."""
+
+  @classmethod
+  def setUpClass(cls):
+    """Internal method to set up the TestCase on a specific storage."""
+    cls.driver = 'overlay2'
+    cls.docker_directory_path = os.path.join('test_data', 'docker')
+    if not os.path.isdir(cls.docker_directory_path):
+      docker_tar = os.path.join('test_data', 'vols.v2.tgz')
+      tar = tarfile.open(docker_tar, 'r:gz')
+      tar.extractall('test_data')
+      tar.close()
+    cls.explorer_object = explorer.Explorer()
+    cls.explorer_object.SetDockerDirectory(cls.docker_directory_path)
+
+    cls.driver_class = storage.Overlay2Storage
+    cls.storage_version = 2
+
+  @classmethod
+  def tearDownClass(cls):
+    shutil.rmtree(cls.docker_directory_path)
+
+  def testGenerateBindMountPoints(self):
+    """Tests generating command to mount 'bind' MountPoints."""
+    self.maxDiff = None
+    de_object = de.DockerExplorerTool()
+    de_object._explorer = self.explorer_object
+    container_obj = de_object._explorer.GetContainer(
+        '8b6e90cc742bd63f6acb7ecd40ddadb4e5dee27d8db2b739963f7cd2c7bcff4a')
+
+    commands = container_obj.storage_object._MakeVolumeMountCommands(
+        container_obj, '/mnt')
+    commands = [' '.join(x) for x in commands]
+    expected_commands = [
+        (
```

Comment:
suggestion for readability:
either remove the extra line and make it `('/bin...` directly or have the opening and closing parentheses on the same indention level, ex:
(
    `line1`
    `line2`
).format(os....)

Reply:
done